public/kibana-integrations/discover/application/angular/directives/histogram.tsx

```diff
@@ -315,7 +313,7 @@ export class DiscoverHistogram extends Component<DiscoverHistogramProps, Discove
         />
         <LineAnnotation
           id="line-annotation"
-          domainType={AnnotationDomainTypes.XDomain}
+          domainType="xDomain"
```

Comment:
Could be on constant

Reply:
This constant changes between 7.14 and 7.13, to maintain compatibility I replace it with a string.